charms/istio-pilot/src/manifests/auth_filter.yaml.j2

```diff
@@ -6,6 +6,7 @@ metadata:
   namespace: {{ envoyfilter_namespace }}
 spec:
   configPatches:
+  {% for gateway_port in gateway_ports %}
```

Comment:
Is it possible for `portNumber` to have more than one value? How would this look like when rendered?

Reply:
I think [this conversation](https://github.com/canonical/istio-operators/pull/277#discussion_r1212891463) will apply here too